### DISCUSSION TITLE
Error when creating microservice from Vonage template

### DISCUSSION THREAD

**Discussion by pietrolerro** [mia-platform/community#126] [Q&A] [answered]
Hello! 🖖🏻

I'm trying to create a microservice starting from the vonage template, it is giving me an error. 
It does create the repository, but it is empty.

Any advice ?

Attached: the response from chrome devTools' network tab
<img width="364" alt="image (3)" src="https://user-images.githubusercontent.com/38533139/204859323-96fd647e-e414-4edf-830d-02fb73d3fc6d.png">
<img width="345" alt="image (4)" src="https://user-images.githubusercontent.com/38533139/204859339-ec551521-5013-4824-9954-26a209090843.png">

**Comment by fredmaggiowski** [comment]
Hi Pietro, we noticed a bug early this morning happening when there are certain file in the original repository; this bug is affecting a few marketplace items and we're working on a fix. 

We'll let you know as soon as the bugfix is up; in the meantime if you want to try the Vonage template you can find it here: https://github.com/nexmo-se/video-express-react-app

**Reply by pietrolerro** [reply]
Thank you Federico :) 
I'll try this out, 
have a great day ^^

**Reply by fredmaggiowski** [reply]
You too; i'll keep you updated as soon as the fix will be out

**Reply by fredmaggiowski** [reply, marked as answer]
Hey @pietrolerro sorry for the delay, we've released yesterday Mia-Platform v10.2.0 that features the bugfix for this issue, therefore you should now be able to create the repository. If you are interested you can find the release note [here](https://next.docs.mia-platform.eu/docs/release-notes/v10.2.0).

**Reply by pietrolerro** [reply]
Awesome! Thank you!
